src/plugins/vis_type_timelion/server/series_functions/es/index.js

```diff
@@ -102,10 +102,11 @@ export default new Datasource('es', {
     );
 
     const scriptedFields = indexPatternSpec?.getScriptedFields() ?? [];
+    const runtimeFields = indexPatternSpec?.getComputedFields().runtimeFields ?? {};
```

Comment:
Can we get scripted and runtime fields using getComputedFields() so that we don't call getScriptedFields?